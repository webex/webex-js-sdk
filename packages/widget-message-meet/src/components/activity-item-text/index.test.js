import React from 'react';
import renderer from 'react-test-renderer';

import ActivityItemText from '.';

describe(`ActivityItemText component`, () => {
  const content = `Test Text`;

  const component = renderer.create(
    <ActivityItemText content={content} />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
