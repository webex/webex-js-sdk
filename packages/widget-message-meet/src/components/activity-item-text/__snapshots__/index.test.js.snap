exports[`ActivityItemText component renders properly 1`] = `
<div
  className="activity-text"
  dangerouslySetInnerHTML={
    Object {
      "__html": "Test Text",
    }
  } />
`;
